planter/planter.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
no. this script in particular should work with any posix shell and doesn't assume you have bash installed to /bin/bash  because it runs on dev machines linux/mac.